src/Newbe.BookmarkManager/Components/ControlPanel.razor

```diff
@@ -221,4 +226,35 @@
         await LoadUserOptions();
     }
 
+    private Task OnChangePinyinFeatureAccessToken(string accessToken)
+    {
+        try
+		{
+			_modal.PinyinFeature.Expire = WebHelper.GetJwtExp(accessToken);
+		}
+		catch (Services.Common.AccessTokenInvalidException e)
+		{
+			_modal.PinyinFeature.Expire = null;
+			_modal.PinyinFeature.AccessToken = string.Empty;
+			throw;
```

Comment:
It is no need to re-throw exception since there is no thing you can do if this exception catched by monitor tool.
Only throw exception if your think it should be collected by monitor tool like Application Insight.